libfuse/folderlist.go

```diff
@@ -146,6 +151,11 @@ func (fl *FolderList) Mkdir(ctx context.Context, req *fuse.MkdirRequest) (_ fs.N
 	fl.fs.log.CDebugf(ctx, "FL Mkdir")
 	tlfName := libkbfs.CanonicalTlfName(req.Name)
 	defer func() { fl.reportErr(ctx, libkbfs.WriteMode, tlfName, err) }()
+	if strings.HasPrefix(req.Name, "._") {
```

Comment:
I think the `._`s are always files and never directories, no matter the thing it corresponds to is a directory or a file.

(same for `fs.go`)